templates/base.twig

```diff
@@ -49,6 +49,8 @@
 
 		{{ function('wp_footer') }}
 	{% endblock %}
-	<script id="countries_script" type="application/json">{% include "countries.json" %}</script>
+	{% if not new_design_country_selector %}
+		<script id="countries_script" type="application/json">{% include "countries.json" %}</script>
```

Comment:
A good option is to set as `async` to this `script`. This allows to stop the HTML parsing only when the script is executed.